testing/src/main/java/org/oppia/testing/screenshots/ScreenshotManager.kt

```diff
@@ -0,0 +1,69 @@
+package org.oppia.testing.screenshots
+
+import android.app.Activity
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.os.Environment
+import android.view.View
+import java.io.File
+import java.io.FileOutputStream
+
+const val OUTPUT_FOLDER_NAME = "OppiaScreenshots"
+
+/**A class that is used to take screenshots from some activity.*/
+class ScreenshotManager {
+
+  /**
+   * A function that takes a screenshot of a given activity and stores it in the
+   * application's files.
+   * @param activity The target activity to take a screenshot of.
+   * @return a bitmap representing the screenshot of the activity.
+   */
+  fun takeScreenshot(activity: Activity): Bitmap {
+    val view = activity.window.decorView.rootView
```

Comment:
Can you write one test case for `HintsAndSolution` or `ConceptCard` too?

Reply:
Done